### PR TITLE
Add new tags

### PR DESCRIPTION
**Pull request by EugeneGohh**
### Description

Add `NFT` and `Blockchain` as a part of tags.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by Dusch4593** [comment]
Thank you for the PR, @EugeneGohh! I've been wondering about this for a bit and am wondering if "NFT" and "Blockchain" might be too specific? Tags are intended to be used for recommending related entries, so I'm wondering if "Cryptography" is more suitable since NFTs, Bitcoin, Blockchains/ledgers, etc. pertain to it?

cc. @sonnynomnom @KTom101 @yangc95

**Comment by SSwiniarski** [comment]
@Dusch4593, this PR refers back to the tags added to [473](https://github.com/Codecademy/docs/pull/473) which I just reviewed. Do you think they should be in the same PR? Also, IMHO, I agree that NFT might be too specific, but "blockchain" seems a reasonable tag.

**Comment by Dusch4593** [comment]
@SSwiniarski Let's settle this PR before merging #473. I don't mind "Blockchain" being added to `tags.md`. I only recommend just adding  "cryptography" because blockchains are blocks of cryptographic hashes strung together. (I.e. I think blockchain is more specific than cryptography).

File: documentation/tags.md [hunk 1]
```diff
@@ -172,6 +173,7 @@ MySQL
 Naive Bayes
 Natural Language Processing
 Netlify
+NFT
```
Comment (Dusch4593) [inline documentation/tags.md:176]: I recommend replacing with Cryptocurrency or Cryptography.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@EugeneGohh NFTs are huge ❗ Since this PR is a byte-sized change I'm going to merge it with just one review approval 👍

**Pull request merged**